pkg/controller/node/BUILD

```diff
@@ -38,6 +38,7 @@ go_library(
         "//pkg/cloudprovider:go_default_library",
         "//pkg/cloudprovider/providers/gce:go_default_library",
         "//pkg/controller:go_default_library",
+        "//pkg/kubelet:go_default_library",
```

Comment:
Yeah, this is not acceptable (in the sense that kubelet is going to eventually move out of tree).  If you want to share logic, make a new package not in pkg/kubelet (or underneath it)